metecho/api/serializers.py

```diff
@@ -171,18 +171,21 @@ class RepoPermissionSerializer(serializers.Serializer):
     admin = serializers.BooleanField(required=False)
 
 
-class GitHubUserSerializer(serializers.Serializer):
+class GitHubUserMinimalSerializer(serializers.Serializer):
     id = serializers.CharField()
     login = serializers.CharField()
-    name = serializers.CharField(required=False)
     avatar_url = serializers.URLField()
+
+
+class GitHubUserSerializer(GitHubUserMinimalSerializer):
+    name = serializers.CharField(required=False)
```

Comment:
Do we not get the `name` when we check members of an org?

Reply:
I'll check and add it to the minimal serializer if available

Reply:
Not a big deal, the code is written to fall back to username if name is not available.

Oh, it looks like we're manually expanding users with this extra info in [the other use-case](https://github.com/oddbird/Metecho/blob/main/metecho/api/jobs.py#L708-L719). Sounds like this is sort of related to [my other comment](https://github.com/oddbird/Metecho/pull/1376#discussion_r870626097), and these two blocks of code might be combined/shared?

Reply:
Just confirmed the members at this point are instances of [`ShortUser`](https://github3py.readthedocs.io/en/master/api-reference/users.html#github3.users.ShortUser), and its missing the `name`.